core/src/com/unciv/logic/trade/TradeEvaluation.kt

```diff
@@ -235,6 +251,13 @@ class TradeEvaluation {
             }
         }
     }
+    fun distanceDiscount(civInfo: CivilizationInfo, capitalcity: CityInfo, city: CityInfo): Int{
+
+        if (abs(capitalcity.location.x - city.location.x) > 500 && abs(capitalcity.location.x - city.location.x) > 500){ // calculates distance between cities
+            return min(50, capitalcity.getTiles().toList()[0].aerialDistanceTo(city.getTiles().toList()[0]) * civInfo.getEraNumber())
```

Comment:
????
Why are you taking ALL the tiles and then taking the first?
If you want the center tile, city.getCenterTile()

Reply:
fixed